src/main/java/org/pih/petl/job/schedule/PetlScheduledExecutionTask.java

```diff
@@ -49,20 +50,21 @@ public void execute(JobExecutionContext jobExecutionContext) throws JobExecution
                 log.debug("Found " + jobs.size() + " configured Jobs");
                 for (String jobPath : jobs.keySet()) {
                     log.debug("Checking job: " + jobPath);
-                    JobExecution latestExecution = etlService.getLatestJobExecution(jobPath);
-                    if (latestExecution == null) {
-                        log.info("Job: " + jobPath + " - Executing for the first time.");
-                        etlService.executeJob(jobPath);
-                    }
-                    else {
-                        Date lastStartDate = latestExecution.getStarted();
-                        log.debug("Last Execution Start: " + lastStartDate);
-                        Date lastEndDate = latestExecution.getCompleted();
-                        log.debug("Last Execution End: " + lastEndDate);
-                        if (lastEndDate != null) {
-                            PetlJobConfig jobConfig = jobs.get(jobPath);
-                            Schedule schedule = jobConfig.getSchedule();
-                            if (schedule != null) {
+                    PetlJobConfig jobConfig = jobs.get(jobPath);
```

Comment:
The purpose of this change is to only run a job if that job has a schedule defined.  This is necessary in order for the "job-pipeline" job type to work correctly, as we typically only want the jobs that it refers to to be executed within the job-pipeline, not as independent jobs.